dashbuilder/dashbuilder-runtime/src/main/java/org/dashbuilder/shared/model/DashbuilderRuntimeMode.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dashbuilder.shared.model;
+
+/**
+ * Execution for dashbuilder Runtime
+ *
+ */
+public enum DashbuilderRuntimeMode {
+
+    /**
+     * Used when only the first import is used
+     */
+    STATIC,
+
+    /**
+     * Allow users to import once
+     */
+    SINGLE_IMPORT,
+
+    /**
+     * Users can import how many dashboards they want and reopen them later.
+     */
+    MULTIPLE_IMPORT;
```

Comment:
William, I think we should remove this option at this stage and add it later if necessary.